contracts/src/core/libraries/v3/access-control/AccessControlConstants.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+library AccessControlConstants {
+    bytes32 internal constant OWNER_ROLE = keccak256("OWNER");
```

Comment:
iiuc this is only the OWNER of the EigenDAAccessControl itself right? How would you set owner roles for other contracts?

Reply:
I see. This should be documented above OWNER_ROLE then

Reply:
IMO, there is no problem with allowing multiple owners. It is to the discretion of the contract owner how many of any role there are. What the contract provides is simply a flexible, transparent, and easily understood mechanism for managing roles.

If we want only a single address for a given key, we might just leverage the address directory instead.